articles/azure-monitor/app/cloudservices.md

```diff
@@ -131,7 +131,38 @@ In Visual Studio, configure the Application Insights SDK for each cloud app proj
 1. Set the *ApplicationInsights.config* file to be copied always to the output directory.  
     A message in the *.config* file asks you to place the instrumentation key there. However, for cloud apps, it's better to set it from the *.cscfg* file. This approach ensures that the role is correctly identified in the portal.
 
-#### Run and publish the app
+## Set up Status Monitor to collect full SQL Queries (optional)
+
+This step is only needed if you want to capture full SQL queries on .NET Framework. 
+
+1. In \*.csdef file Add [startup task](https://docs.microsoft.com/en-us/azure/cloud-services/cloud-services-startup-tasks) for each role similar to 
```

Comment:
removing hardcoded en-us to allow the link to auto-localize based on customer location.

```suggestion
1. In `\*.csdef` file Add [startup task](https://docs.microsoft.com/azure/cloud-services/cloud-services-startup-tasks) for each role similar to 
```